lib/vmfloaty/abs.rb

```diff
@@ -247,8 +247,8 @@ def self.retrieve(verbose, os_types, token, url, user, options, _ondemand = nil)
   #
   # We should fix the ABS API to be more like the vmpooler or nspooler api, but for now
   #
-  def self.translated(res_body)
-    vmpooler_formatted_body = {}
+  def self.translated(job_id, res_body)
```

Comment:
I suggest keeping `res_body` as the first param. It's really easy to break clients that might be using the gem as a library here (which is intended with this tool). So updating a new param as the first position can result in some confusing behavior.